lib/iris/analysis/__init__.py

```diff
@@ -562,6 +555,258 @@ def post_process(self, collapsed_cube, data_result, **kwargs):
 
         return collapsed_cube
 
+    def aggregate_shape(self, **kwargs):
+        """
+        The shape of the additive dimension created by the aggregator.
```

Comment:
new term  ??additive dimension?? sounds a bit weird.
Could we just say "The shape of the extra dimensions added by the aggregator" ?
Do I understand that these always appear at the front ? 
I think it would be useful if the wording generally could make that clearer -- at present it sounds like you could have something more complex.